Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -420,6 +420,10 @@ public void generateShortUrl(@NonNull Context context, @NonNull LinkProperties l
     //------------------ Share sheet -------------------------------------//
 
     public void showShareSheet(@NonNull Activity activity, @NonNull LinkProperties linkProperties, @NonNull ShareSheetStyle style, @Nullable Branch.BranchLinkShareListener callback) {
+       showShareSheet(activity, linkProperties, style, callback, null);
+    }
+
+    public void showShareSheet(@NonNull Activity activity, @NonNull LinkProperties linkProperties, @NonNull ShareSheetStyle style, @Nullable Branch.BranchLinkShareListener callback, Branch.IChannelProperties iChannelProperties) {
```

Comment:
Nit pick ,can we change the variable name `iChannelProperties` to `channelProperties` callback